api/http/server.go

```diff
@@ -255,6 +260,9 @@ func (s *Server) HandlePostRaw(w http.ResponseWriter, r *http.Request) {
 		toEncrypt = true
 	}
 
+	// Set the pinCounter if there is a pin header present in the request
+	headerPin := r.Header.Get(pin.SwarmPinHeaderName)
```

Comment:
Because declaring in server.go will look out of place. All things related to pin should be in pin package. See the usage in server.go.. it looks like "pin.HeaderName" which is nice i think.